DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -143,58 +151,15 @@ ILogger<RegistrationService> logger
             return (candidateNumber, delegateRegistrationModel.Approved);
         }
 
-        public string RegisterDelegateByCentre(DelegateRegistrationModel delegateRegistrationModel, string baseUrl)
-        {
-            using var transaction = new TransactionScope();
-
-            var candidateNumber = CreateAccountAndReturnCandidateNumber(delegateRegistrationModel);
-
-            if (delegateRegistrationModel.PasswordHash != null)
-            {
-                passwordDataService.SetPasswordByCandidateNumber(
-                    candidateNumber,
-                    delegateRegistrationModel.PasswordHash
-                );
-            }
-            else if (delegateRegistrationModel.NotifyDate.HasValue)
-            {
-                passwordResetService.GenerateAndScheduleDelegateWelcomeEmail(
-                    delegateRegistrationModel.Email,
-                    candidateNumber,
-                    baseUrl,
-                    delegateRegistrationModel.NotifyDate.Value,
-                    "RegisterDelegateByCentre_Refactor"
-                );
-            }
-
-            var supervisorDelegateRecordIdsMatchingDelegate =
-                GetPendingSupervisorDelegateIdsMatchingDelegate(delegateRegistrationModel).ToList();
-
-            // We know this will give us a non-null user.
-            // If the delegate hadn't successfully been added we would have errored out of this method earlier.
-            var delegateUser = userDataService.GetDelegateUserByCandidateNumber(
-                candidateNumber,
-                delegateRegistrationModel.Centre
-            )!;
-
-            if (supervisorDelegateRecordIdsMatchingDelegate.Any())
-            {
-                supervisorDelegateService.AddDelegateIdToSupervisorDelegateRecords(
-                    supervisorDelegateRecordIdsMatchingDelegate,
-                    delegateUser.Id
-                );
-            }
-
-            transaction.Complete();
-
-            return candidateNumber;
-        }
-
-        public void RegisterCentreManager(AdminRegistrationModel registrationModel, int jobGroupId)
+        public void RegisterCentreManager(
+            AdminRegistrationModel registrationModel,
+            int jobGroupId,
+            string? professionalRegistrationNumber

```

Comment:
We can move this inside RegistrationModel I think. Eventually all registrations will provide a PRN.

Reply:
I've moved this in, but I wouldn't be surprised if this conflicts with some of @JBsoftwire's in progress work on the self registration